exercises/practice/affine-cipher/.meta/config.json

```diff
@@ -0,0 +1,22 @@
+{
+  "authors": [],
```

Comment:
Not a Haskell maintainer here, but this should contain your GitHub username.

Reply:
You're right, corrected.